.gitlab-ci.yml

```diff
@@ -20,7 +20,7 @@ cache:
 
 # this job installs NPM dependencies and Cypress
 install:
-  image: cypress/base:latest
+  image: cypress/base/18.14.1
```

Comment:
I think these all need to be `cypress/base:18.14.1` since `18.14.1` is a tag name

Reply:
@astone123 

> I think these all need to be cypress/base:18.14.1 since 18.14.1 is a tag name

You are absolutely right! Thank you for catching my mistake. I will correct it.

Reply:
The reason for my slip up is that the link to the container on the Docker hub uses a slash delimiter between `base` and `18.14.1` and when I was writing I failed to switch mental modes 🙁 

https://hub.docker.com/layers/cypress/base/18.14.1/images/sha256-a63a2bec81efa794970664ad24466c1627138b8668d37f6cd22fd71f6080caad

Reply:
I have amended the original commit because the commit message also had the wrong syntax. https://github.com/cypress-io/cypress-example-kitchensink/pull/614/commits/782c6886a173b14e1777f80585cf04660a05e415 is the replacement.